plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/EvaluateSuggestion.php

```diff
@@ -58,6 +65,12 @@ public static function evaluate( $spec, $logger_args = array() ) {
 	 * @return array The visible suggestions and errors.
 	 */
 	public static function evaluate_specs( $specs, $logger_args = array() ) {
+		$specs_key = md5( wp_json_encode( $specs ) );
```

Comment:
Nit: With PHP 8.1, `xxh3` is the most performant hash operation. Perplexity suggested a utility function that adapts to the algorithm or fallback to `crc32` if it's not available. Since this may run around 20 times per request, I think the performance improvements may be worth.

<img width="500" alt="image" src="https://github.com/user-attachments/assets/6b943bd3-1477-4407-8919-6756bbba4f17">

More info: https://xxhash.com/

Following is a generated code (untested):

```php
function hash_data($data) {
    if (function_exists('hash') && in_array('xxh3', hash_algos())) {
        // Use xxHash (xxh3) if available
        return hash('xxh3', $data);
    } else {
        // Fall back to CRC32
        return (string)crc32($data);
    }
}
```

Reply:
Good suggestion. I've updated the code as suggested in df27eaac12ee13e2ae014297be001f0832e3f75e